docker-app/qfieldcloud/core/views/accounts_views.py

```diff
@@ -0,0 +1,33 @@
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
+from django.contrib import messages
+from django.utils.translation import gettext as _
+from allauth.account.models import EmailAddress
+
+
+def resend_confirmation_email(request: HttpRequest) -> HttpResponse:
+    """
+    Resends a confirmation email to a new unverified user.
+    """
+    if request.method != "POST":
+        return HttpResponseRedirect(request.headers.get("referer", "account_login"))
+
+    email_addr = request.session.get("account_verified_email")
```

Comment:
```suggestion
    email_address = request.session.get("account_verified_email")
```
We have live with 3 bytes more.